### PR TITLE
Added Blogs Route (SuperUser)

### PR DESCRIPTION
**Pull request by greykoalacode**
## Description of the change

> Superuser route to view blogs has been added and feature of pinning the blog has been updated. Do let me know if any changes!

## Type Of Change
- [ ] Bug Fix (non-breaking change that fixes an issue)
- [*] New Feature (non-breaking change that adds functionality)
- [ ] Breaking Change (fix or feature that would cause existing functionality to not work as expected)

## Related Issues

> Fix [#<ISSUE_NUMBER>](<ISSUE_LINK>) 

## Development & Code Review 
- [*] Fixed all the Eslint errors (run `npm run lint-fix`)
- [*] All ESLint rules passes locally (run `npm run lint`)
- [*] The code changed/added has been covered with tests
- [*] All the test cases added/changed are passing

### CODE REVIEW

File: src/Components/superuser/blogs/blogsList/index.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import useActivePageState from '../../../../customHooks/useAcitvePageState';
+import CustomSnackbar from '../../../common/Snackbar/index';
+import LoadingCardArray from '../../../common/LoadingCardArray';
+
+
+// All components are mostly same, fetched like the way you did
+// only change is in the blog card. added a button and
+// made it EnhancedBlogCard
+
+const AllBlogsPageContainer = () => {
```
Comment (ac030540) [inline src/Components/superuser/blogs/blogsList/index.jsx:19]: There is no need to create a separate component at all for this. What you can do is, pass a prop named isSuperuserRoute to index.jsx of blogList in drawer and pass the same prop to BlogCard component. Using this prop you can conditionally render the data inside the components.

File: src/Components/drawer/blogs/blogsList/index.jsx [hunk 2]
```diff
@@ -11,7 +12,7 @@ import useActivePageState from '../../../../customHooks/useAcitvePageState';
 import CustomSnackbar from '../../../common/Snackbar/index';
 import LoadingCardArray from '../../../common/LoadingCardArray';
 
-const AllBlogsPageContainer = () => {
+const AllBlogsPageContainer = ({ isSuperuserRoute }) => {
```
Comment (ac030540) [inline src/Components/drawer/blogs/blogsList/index.jsx:15]: Set the default value for isSuperuserRoute. Make it `isSuperuserRoute=false`.

Comment (ac030540) [reply src/Components/drawer/blogs/blogsList/index.jsx:15]: Do the change in all the components where you have defined it.

Comment (greykoalacode) [reply src/Components/drawer/blogs/blogsList/index.jsx]: Yes I've done it, check it

**Pull request merged**